wizeline/wize-q-remix
# revert: publish anonymous questions (previous version)

#### What does this PR do?
Revert all the changes related with the previous version, related with the anonymous questions: 
- _Select a person according to the department to create an anonymous question_
- _[admin] publish an anonymous question._
- _Send an alert to the Slack channel until and admin publish an anonymous question_
- _Show the alert with the link of the question and the advice to save it._


⚠️ _The option to send emails is still available._ 

#### Where should the reviewer start?
- Verify that the reverts commits only 'reverts' the changes related with publish an anonymous question.

#### How should this be manually tested?
1.  Login as [admin](https://github.com/wizeline/wize-q-remix/wiki/How-to-give-admin-permissions-to-my-user). 
2.  Double check that you can not see the option to publish a question (anonymous) 
3. When creating an anonymous question, verify that it is not necessary to select a person according to the department.
4. Verify, after posting a question, that the alert to save the question link is not displayed.
5. Double check that everything else is working correctly.
6. :tada:

#### Any background context you want to provide?
We have to revert the changes related with the functionality to publish an anonymous question. 
Now the stakeholders don't want to have the anonymous question private, they want that those questions appears immediately in the home page (like in production) 

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Closes #x

#### Screenshots
* Crete a question 

https://user-images.githubusercontent.com/97203617/231308287-b3ed5bcc-ba9b-404d-a102-83721432c6c2.mov

* Publish Option is no available.
<img width="673" alt="Screen Shot 2023-04-11 at 17 17 57" src="https://user-images.githubusercontent.com/97203617/231308825-6104b409-a9a2-4bec-8835-9732aceeb2ff.png">
